src/ViewInstance.js

```diff
@@ -563,6 +600,18 @@ class ViewInstance extends React.Component {
                 </Icon>
               </Button>
             )}
+            <Button
+              id="quick-export-trigger"
+              onClick={() => {
+                onToggle();
+                this.triggerQuickExport();
```

Comment:
@UladzislauKutarkin please add `//NOSONAR` 

`this.triggerQuickExport(); //NOSONAR` and comment above that triggerQuickExport has try/catch inside and doesn't require catch or await here

Reply:
@NikitaSedyx I fixed sonar by wrapping that function